README.md

```diff
@@ -173,13 +173,13 @@ go test -v ./...
 
 You can set the following environment variables to change the behavior of knuu:
 
-| Environment Variable        | Description                                        | Possible Values                  | Default  |
-|-----------------------------|----------------------------------------------------|----------------------------------|----------|
-| `KNUU_TIMEOUT`              | The timeout for the tests.                         | Any valid duration               | `60m`    |
-| `KNUU_NAMESPACE`            | The namespace where the instances will be created. | Any valid namespace name         | `test`   |
-| `KNUU_BUILDER`              | The builder to use for building images.            | `docker`, `kubernetes`           | `docker` |
-| `KNUU_DEDICATED_NAMESPACE`  | Creates a dedicated namespace for the test.        | `true`, `false`                  | `false`  |
-| `DEBUG_LEVEL`               | The debug level.                                   | `debug`, `info`, `warn`, `error` | `info`   |
+| Environment Variable        | Description                                                                                                                                                                                                                                  | Possible Values                  | Default  |
+|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------|
+| `KNUU_TIMEOUT`              | The timeout for the tests.                                                                                                                                                                                                                   | Any valid duration               | `60m`    |
+| `KNUU_NAMESPACE`            | The namespace where the instances will be created.                                                                                                                                                                                           | Any valid namespace name         | `test`   |
+| `KNUU_BUILDER`              | The builder to use for building images.                                                                                                                                                                                                      | `docker`, `kubernetes`           | `docker` |
+| `KNUU_DEDICATED_NAMESPACE`  | Creates a namespace for the test, and it gets deleted after `KNUU_TIMEOUT`.<br/> *This ENV var has priority over `KNUU_NAMESPACE`*.<br/>The namespace uses a prefix (`knuu-`) + the unique id as name, example: `knuu-20240404-102053-681`   | `true`, `false`                  | `false`  |
```

Comment:
It's not clear what 'unique id` means here.

Also, as this explanation is quite long. Maybe have a quick explanation in the table and then reference to a note below the table to keep the table short and readable.